pstore/kafka/backoff_test.go

```diff
@@ -0,0 +1,32 @@
+package kafka
+
+// TODO: If optiopay/kafka accepts pull request, remove this file.
```

Comment:
BTW you might consider looking at our fork of the kafka module. We've run into the same issues you did and have been working to improve the throughput and functionality: https://github.com/dropbox/kafka

We've entirely rewritten the network handling layer since Kafka has another issue where it can only handle a single request at a time per TCP connection, so we've had to add connection pooling. We've also redone how the mutex usage works to avoid the locking problem you ran into before.